ghost/core/test/e2e-api/admin/invites.test.js

```diff
@@ -224,32 +224,5 @@ describe('Invites API', function () {
                 .expect('Cache-Control', testUtils.cacheRules.private)
                 .expect(403);
         });
-
-        it('Can add a new invite by API Key with the Contributor Role', async function () {
```

Comment:
[To help reviewers] Removed duplicated test (exactly same code as [this one](https://github.com/TryGhost/Ghost/blob/4598a342f030b2899c18fb4b699495504953c3f2/ghost/core/test/e2e-api/admin/invites.test.js#L188-L213));